include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -141,6 +141,14 @@ Operator<dim, Number>::fill_matrix_free_data(MatrixFreeData<dim, Number> & matri
   if(param.body_force)
     matrix_free_data.append_mapping_flags(BodyForceOperator<dim, Number>::get_mapping_flags());
 
+  if(param.problem_type == ProblemType::Unsteady)
+  {
+    // The boundary mass operator is only required, if we have damping terms from the Robin
+    // condition, i.e., in the unsteady case.
+    matrix_free_data.append_mapping_flags(
+      BoundaryMassOperator<dim, Number, dim /* n_components */>::get_mapping_flags());
+  }
```

Comment:
Similar to this comment https://github.com/exadg/exadg/pull/555/files#r1457203165: I think we do not always have `Robin` type BCs, so I would expect an if-statement here like `if(robin){...}`?